js/layout-javascript/news-feed-code.js

```diff
@@ -247,7 +256,11 @@ DynamicList.prototype.attachObservers = function() {
         }
       });
     })
-    .on('click', '.news-feed-detail-overlay-close, .news-feed-detail-overlay-screen', function(event) {
+    .on('click keydown', '.news-feed-detail-overlay-close, .news-feed-detail-overlay-screen', function(event) {
+      if (event.type !== 'click' && event.keyCode !== 32 && event.keyCode !== 13) {
```

Comment:
Why isn't this updated too?

```suggestion
      if (!_this.Utils.Event.isExecute(event)) {
```

Reply:
@tonytlwu removed